SPECS/docbook-style-dsssl/docbook-style-dsssl.spec

```diff
@@ -66,6 +66,10 @@ fi
 exit 0
 
 %changelog
+* Fri Sep 16 2022 Osama Esmail <osamaesmail@microsoft.com> - 1.79-31
+- Moved from SPECS-EXTENDED to SPECS
+- License verified
```

Comment:
Ideally, we should also mention the license through the `%license` macro. If the project doesn't include any file explicitly dealing with the license, then we create a `LICENSE.PTR` file included as one of the sources, pass that to the `%license` macro and inside the file we explain how we figured out what the licensing is. See `numad.spec` for an example of this approach.

Reply:
So, docbook-style-dsssl has a license in its README that I was able to use in LICENSE.PTR, but docbook-dtds doesn't hae any license information in its source code. docbook-dtd-xml is the package most similar to docbook-dtds and also doesn't have a %license macro, so should I just not use on for docbook-dtds?